pkg/investigations/clustermonitoringerrorbudgetburn/metadata.yaml

```diff
@@ -10,3 +10,7 @@ rbac:
       resources:
         - clusteroperators
 customerDataAccess: false
+sop: https://github.com/openshift/ops-sop/blob/master/v4/troubleshoot/clusteroperators/monitoring.md
+issues:
+  - issue: https://issues.redhat.com/browse/OCPBUGS-33863
+    removeAfter: 4.17
```

Comment:
IMO this should be in a README of the investigation package. 

@MateSaary maybe we could generate a README as part of the boilerplate setup as well: https://github.com/openshift/configuration-anomaly-detection/pull/361 - thoughts?

Reply:
i like the idea of having a readme for each investigation  and we start of from there
 once we implement some kind of lifecycle automation ( PR checks tracking the bugs) we can revisit this